### PR TITLE
docs(reporters): Update readme with new reporters options

### PR DESCRIPTION
**Pull request by Mobrockers**

### CODE REVIEW

File: src/Stryker.CLI/README.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ All available loglevels are:
...
+#### Mutation testrun reporters 
+
+`dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']`
+
+All available reporter options are:
+ * All - This will enable all available reporters
+ * ConsoleProgressBar - This reporter will display the progress of the mutation testrun as a progress bar
+ * ConsoleProgressDots - This reporter will display the progress of the mutation testrun as a dot progress bar in the format .$$$...$
+ * ConsoleReport - At the end of the mutation testrun, this reporter will display a styled report in a console window
+ * Json - At the end of the mutation testrun, this reporter will output the testresult as a json object which can be used as input to a html report
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]: Maybe it would be nice to mark Progress and ConsoleReport as `(default)`?

**Comment by Mobrockers** [comment]
I'll add examples of the output before merging

File: src/Stryker.CLI/README.md [hunk 2]
```diff
@@ -54,6 +54,53 @@ All available loglevels are:
 
 `dotnet stryker --max-concurrent-test-runners`
 
+#### Mutation testrun reporters 
+
+`dotnet stryker --reporters ['ConsoleReport', 'ConsoleProgressBar']`
+
+All available reporter options are:
+ * All - This will enable all available reporters
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]: It seems unlogical to start with all. Readers won't know what all reporters are when first reading this.

File: src/Stryker.CLI/README.md [hunk 3]
```diff
@@ -54,6 +54,53 @@ All available loglevels are:
...
+ * ConsoleProgressBar - This reporter will display the progress of the mutation testrun as a progress bar
+ Example output:
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]: `Example output:` could use an extra newline. Now it is displayed right behind the ConsoleProgressBar explaination.

```suggestion

Example output:
```

File: src/Stryker.CLI/README.md [hunk 4]
```diff
@@ -54,6 +54,53 @@ All available loglevels are:
...
+```
+Tests progress | █████████  | 14 / 15 | 94 % | ~0m 07s |
+Killed : 1
+Survived: 12
+Timeout : 1
+```
+ * ConsoleProgressDots - This reporter will display the progress of the mutation testrun as a dot progress bar in the format .STS...S
+ Example output:
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]:
```suggestion

Example output:
```

File: src/Stryker.CLI/README.md [hunk 5]
```diff
@@ -54,6 +54,53 @@ All available loglevels are:
...
+[Timeout] Binary expression mutation on line 11: '1 < 0' ==> '1 > 0'
+[Survived] Binary expression mutation on line 11: '1 < 0' ==> '1 <= 0'
+--- StringMagic.cs [0/4 (0.00 %)]
+[Survived] Binary expression mutation on line 7: 'first.Length > 2' ==> 'first.Length < 2'
+[Survived] Binary expression mutation on line 7: 'first.Length > 2' ==> 'first.Length >= 2'
+[Survived] String mutation on line 18: '"toto"' ==> '""'
+[Survived] Boolean mutation on line 19: 'true' ==> 'false'
+```
+ * Json - At the end of the mutation testrun, this reporter will output the testresult as a json object which can be used as input to a html report
+ Example: See https://github.com/stryker-mutator/mutation-testing-elements/issues/1#issue-339099563
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]:
```suggestion

Example: See https://github.com/stryker-mutator/mutation-testing-elements/issues/1#issue-339099563
```

File: src/Stryker.CLI/README.md [hunk 6]
```diff
@@ -54,6 +54,53 @@ All available loglevels are:
...
+ * All - This will enable all available reporters
+ * ConsoleProgressBar - This reporter will display the progress of the mutation testrun as a progress bar
+ Example output:
+```
+Tests progress | █████████  | 14 / 15 | 94 % | ~0m 07s |
+Killed : 1
+Survived: 12
+Timeout : 1
+```
+ * ConsoleProgressDots - This reporter will display the progress of the mutation testrun as a dot progress bar in the format .STS...S
```
Comment (richardwerkman) [inline src/Stryker.CLI/README.md]: I'm missing an explaination for what each char means.

Comment (Mobrockers) [reply src/Stryker.CLI/README.md]: Same :p
I'm guessing there's **S**urvived **T**imeout and **.**Killed. Are there any other possible values?

Comment (richardwerkman) [reply src/Stryker.CLI/README.md]: Nope that's all 👍

**Pull request merged**